web/src/i18n/locales/ja.json

```
@@ -84,7 +84,10 @@
         "submitterName": "提出者",
         "thanks": "投稿ありがとうございます！",
         "title": "スピーカーを指名する",
-        "topics": "得意分野",
+        "topics": {
+            "label": "得意分野",
+            "hint": "Type anything and press enter to add new topics"
```

```suggestion
            "hint": "何かを入力してEnterキーを押し、新しいトピックを追加します"
```

If we don't know the key, we can either 1) Add machine-translated Japanese from Google Translate (added here as a suggestion) or 2) omit the key from ja.json, as our vue-18n will always fall back to en.json on missing keys.

---

Ah good to know! Thanks!